openhtf/util/threads.py

```diff
@@ -95,7 +95,7 @@ def async_raise(self, exc_type):
     assert self.is_alive(), 'Only running threads have a thread identity'
```

Comment:
Makes me wonder if this assert belongs here. Maybe `assert self.ident is not None` is appropriate, seems like that's actually guaranteed to be true, and not subject to a race condition.

```
assert self.ident is not None, 'Thread is missing identifier since it was never started.'
```